images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -49,16 +49,35 @@ for latest_package in ${LATEST_DOTNET_PACKAGES[@]}; do
 done
 
 # Get list of all released SDKs from channels which are not end-of-life or preview
+latest_5_0_sdk=""
 sdks=()
 for release_url in ${release_urls[@]}; do
     echo "${release_url}"
     releases=$(curl "${release_url}")
     sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdk.version'))
     sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdks[]?' | jq '.version'))
+    if [ "$(echo "${releases}" | jq '."channel-version"')" = "\"5.0\"" ]; then
+        latest_5_0_sdk=$(echo "${releases}" | jq '.["latest-sdk"]')
+    fi
 done
 
 sortedSdks=$(echo ${sdks[@]} | tr ' ' '\n' | grep -v preview | grep -v rc | grep -v display | cut -d\" -f2 | sort -u -r)
 
+#if latest 5.0 sdk is not present in sortedSdks, then add it
+if [ "$latest_5_0_sdk" != "" ] ; then
+    latest_5_0_sdk_found=false
+    for sdk in $sortedSdks
+    do
+        if [ "\"$sdk\"" == "$latest_5_0_sdk" ] ; then
+            latest_5_0_sdk_found=true
+        fi
+    done
+
+    if [ "$latest_5_0_sdk_found" = false ] ; then
+        sortedSdks+=" $(echo $latest_5_0_sdk | cut -d\" -f2)"
+    fi
+fi
```

Comment:
Why do we need this logic? If we merge this PR after release of stable .NET Core 5.0, we don't need this loop and condition. It is enough just adding new url. Please correct me if I am wrong.

The same for others platforms

Reply:
@maxim-lobanov yes thats correct. So the hardcoded urls just need to be appended to the sorted sdks before it goes ga. So can you create the prs for that? I will create separate piublic prs, which we will merge after ga.

Reply:
Sure, we can take care about these changes from our side and abandon your PR if it works for you.

Reply:
ok i will close this one.